### DISCUSSION TITLE
buddhism/breath/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#22] [General]
# buddhism/breath/

https://shanenull.com/buddhism/breath/

**Comment by shane0** [comment]
**Counting the breath meditation in the style of Cormac McCarthy:**

The breath is the measure of all things.

It is the metronome of our lives.

And it is the key to our liberation.

Counting the breath meditation is a simple practice.

Find a comfortable position and close your eyes.

Take a deep breath and count "one" as you inhale.

Exhale and count "two."

Continue counting your breaths all the way to ten.

If your mind wanders, gently bring it back to the breath.

Do not judge yourself. Do not criticize yourself.

Simply observe the breath and count.

Counting the breath meditation is a practice of mindfulness.

It is a practice of paying attention to the present moment.

It is a practice of letting go of the past and the future.

As you count your breaths, you will begin to notice the rhythm of your breath.

You will begin to notice the sensations of breathing.

And you will begin to notice the thoughts and emotions that arise.

Do not judge these thoughts and emotions. Simply observe them and let them go.

The more you practice counting the breath meditation, the more mindful you will become.

You will become more aware of your thoughts and emotions.

And you will become more free from their control.

Counting the breath meditation is a practice of liberation.

It is a practice of breaking the chains of the mind.

It is a practice of finding your true nature.

So sit quietly and count your breaths.

And let the breath lead you to freedom.

**Comment by shane0** [comment]
https://youtu.be/YWT9nULIv2I?si=4-6Kjvu8WEP1uDcW

**Comment by shane0** [comment]
Plum village song
https://youtu.be/A2XdK75HdlM?si=53b47GUGqsc9iCE-

**Comment by shane0** [comment]
Posture
https://youtu.be/-er2gIKj2w8?si=pM7Mt2yQFxJL494f

**Comment by shane0** [comment]
https://youtu.be/9qXu-_sHy5Q?si=F-dDnXzQnroejoxz

**Comment by shane0** [comment]
mindfulness observe consciousness use reflective awareness to discern distraction from practice use breath counting as needed

non judgemental non analyzing
